pkgs/by-name/ty/typescript-go/package.nix

```
@@ -41,7 +43,13 @@ buildGoModule {
   '';
 
   passthru = {
-    updateScript = unstableGitUpdater { };
+    updateScript = _experimental-update-script-combinators.sequence [
+      (unstableGitUpdater { })
+      (nix-update-script {
+        # Updating `vendorHash`
+        extraArgs = [ "--version=skip" ];
```

maybe this works instead of unstableGitUpdater?
```suggestion
        extraArgs = [ "--version=branch" ];
```

---

Thanks for your helpful review! I didn't realize how useful `--version=branch` would be.
Your suggestion really simplified my code.

Updated in https://github.com/NixOS/nixpkgs/compare/7f7965deae7944522855ce019cab85c076e633a0..b484f27c3747860766a168e4cd24f87e5b17d477